frontend/packages/text-editor/src/TextEditor.tsx

```diff
@@ -97,13 +98,26 @@ export const TextEditor = ({
           <Button variant='primary' color='first' onClick={handleAddNewEntryClick} size='small'>
             {t('text_editor.new_text')}
           </Button>
-          <div>
-            <SearchField
-              id='text-editor-search'
-              label={t('text_editor.search_for_text')}
-              value={searchQuery}
-              onChange={handleSearchChange}
-            />
+          <div className={classes.TextEditor__topRow__filter}>
```

Comment:
Should we use camelCase for CSS classes?😄

Reply:
Agree! Unsure how much refactoring to do in this PR tho, since I believe the whole text editor deserves some love 😅

Reply:
Yes, I agree 😄 I think the old classnames should be fixed in another issue, but as this seems like a new change (?), I believe this line should be fixed to improve the code we are editing 😄

Reply:
Hm, yes, I agree, but it will also be unconsistent 🤔 Dont know what is best.. Or less worse haha

Reply:
I would say it should be camelCase because it is already inconsistent 😄 
If we do not make it camelCase, I would say that you should change the classname on line 107 which is ```sortAlphabetically```.